recipe/meta.yaml

```diff
@@ -56,6 +56,8 @@ about:
     PyQt-builder is the PEP 517 compliant build system for PyQt and projects
```

Comment:
SIP_License in the LICENSE file. See also https://www.riverbankcomputing.com/software/sip/
`license: LicenseRef-SIP_License OR GPL-2.0-or-later OR GPL-3.0-or-later`

And setup.py says:
```
# This copy of PyQt-builder is licensed for use under the terms of the SIP
# License Agreement.  See the file LICENSE for more details.
#
# This copy of PyQt-builder may also used under the terms of the GNU General
# Public License v2 or v3 as published by the Free Software Foundation which
# can be found in the files LICENSE-GPL2 and LICENSE-GPL3 included in this
# package.
```

Reply:
thank you, addressed